src/components/UI/Typography/typography.story.tsx

```diff
@@ -21,6 +24,12 @@ export const typography: Story = {
     children: 'Hello World!',
     variant: 'success',
   },
+  // ADD THIS PLAY FUNCTION TO TEST THE COMPONENT
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const textElement = await canvas.getByText('Hello World!');
```

Comment:
Get by text is not stable test, you can add data test id to the typography component and test it by this attribute